localstack-core/localstack/services/apigateway/next_gen/execute_api/gateway.py

```diff
@@ -0,0 +1,33 @@
+from rolo import Response
+from rolo.gateway import Gateway
+
+from . import handlers
+from .context import RestApiInvocationContext
+
+
+class RestApiGateway(Gateway):
+    def __init__(self):
+        super().__init__(context_class=RestApiInvocationContext)
+        self.request_handlers.extend(
+            [
+                handlers.preprocess_request,
+                handlers.method_request_handler,
+                handlers.integration_request_handler,
+                handlers.integration_handler,
+                # temporary handler which executes everything for now
+                handlers.global_temporary_handler,
```

Comment:
nit: Previously we used to use the term "legacy" for this kind of backwards compatibility / migration logic.